.Documentation/articles/innoactive-creator/suspending-interactions.md

```diff
@@ -0,0 +1,5 @@
+# Suspending Interactions
```

Comment:
Don't like suspending at all to be honest. Is not really clear to me. I unfortunately do not have a better wording, but would prefer locking over suspending

Reply:
Disabling VR Interactions?